tests/core/test_config.py

```diff
@@ -324,11 +324,20 @@ def testConfigCustomOverDefault(self, *args):
         self.assertEquals(1, len(checks['initialized_checks']))
         self.assertEquals(2, checks['initialized_checks'][0].instance_count())  # check that we picked the right conf
 
-    @mock.patch('config.get_version', return_value='5.9.1')
+    def tearDown(self):
+        for _dir in self.TEMP_DIRS:
+            rmtree(_dir)
+
+
+class TestManifestValidation(unittest.TestCase):
     def testManifestValidateOK(self, *args):
         manifest_path = '{}/manifest.json'.format(FIXTURE_PATH)
-        validate = validate_sdk_check(manifest_path)
-        self.assertEquals(True, validate)
+        with mock.patch('config.get_version', return_value='5.9.1'):
+            validate = validate_sdk_check(manifest_path)
+            self.assertEquals(True, validate)
+        with mock.patch('config.get_version', return_value='5.12.0'):
```

Comment:
Is the second test to illustrate the tuples are doing their thing versus lexicographical comparison? If so, I think just the second test would suffice.